gtsam/hybrid/HybridBayesNet.h

```diff
@@ -59,6 +59,14 @@ class GTSAM_EXPORT HybridBayesNet : public BayesNet<AbstractConditional> {
    */
   GaussianMixture::shared_ptr atGaussian(size_t i) const;
 
+  /**
+   * @brief Set gaussian mixture at specific index.
+   *
+   * @param i The index to set the gaussian mixture at.
+   * @param gaussian The gaussian mixture shared pointer to add.
+   */
+  void setGaussian(size_t i, const GaussianMixture::shared_ptr &gaussian);
```

Comment:
Why do we have non-const method here? Please re-evaluate.

Reply:
Removed.